### ISSUE TITLE
Document Valkyrie workflows and best-practices

### ISSUE THREAD

**Issue by niccolox** [GetValkyrie/valkyrie#45]
how do they work together?

whats the workflow and best practice?

documentation?

**Comment by neurojavi** [comment]
Yes It would be great to have some documentation about Valkyrie workflows and best-practices.

I've been out of drupal dev for 2 or 3 years and I'm a litle lost. Last site I developed was based on features. I tried hard to use code based dev and I think I achieve it. I had a makefile for the platform in a common git repo I had for all my makefiles, then I made a repo for each feature I used on my site (some general ones that could be reused, and some site specific ones). The makefile contained all modules and features with fixed version number or commit, patches, etc. And I tag the makefiles repo and all used features with a script on each new test or production version that I deployed to my aegir hosting provider.

I had some custom scripts that helped me in creating the platforms and moving sites...

I had to figure all this on my own (I work alone) and since I mainly make projects for free (for educational projects and non profits, or about education) this is a lot of work...

I'm starting some new projects now and have spent one week learning and installing some new tools like Valkyrie... I now have it working and I was about to start working with my old workflow when I saw all this new magic about git in aegir and the drush commands to install sites from git.

So... I think it would be a great help if you share a little about best practices or about your workflow... At least a litle about how to use those dush commands. For example:
drush vsc example.com                  Install a site from the configration provided in the provided git repo

What configuration is drush expecting in the git repo (structure, files...)

Well, I understand that maybe you don't have time to write this docs by now, but maybe a brief intro here, in this issue, or some links to some existing blogs post (if any) or... something, please! :-)

Thanks for making this great tool!

**Comment by ergonlogic** [comment]
We started a docs repo here: https://github.com/GetValkyrie/valkyrie-docs

Much as I prefer to keep docs integrated with the project, making it a separate repo allows us to add additional committers. I'll re-integrate them as a sub-module.

**Comment by ergonlogic** [comment]
I turned off the wiki and issues on the valkyrie-docs project, since they should really stay here in the main project.

**Comment by ergonlogic** [comment]
We still need better documentation of proper workflows, but we now have the valkyrie-docs project as a submodule. We can improve on these within the 0.5.x release cycle.
